src/components/SignName.vue

```diff
@@ -119,7 +129,8 @@ export default {
     computed: {
       form () {
         return {
-          name: this.name,
+          firstName: this.first_name,
```

Comment:
This should be `firstName: this.firstName` and `lastName: this.lastName` since those are what the Vue instance variables are called in `data`

Reply:
Updated Vue instance variables to be the same as they are called in data